lib/block.js

```diff
@@ -3,50 +3,72 @@ import styled, { css } from 'styled-components';
 import { IconButton } from './icon-button';
 import { CodeExample, PropsDefinition, Prop } from './story-utils';
 
+const variants = {
+  // variants for Section
+  info: css`
+    color: var(--colors-primary);
+    background-color: var(--colors-secondaryBackground);
+    border-top: 1px solid var(--colors-border);
+  `,
+  actions: css`
+    color: var(--colors-primary);
+    background-color: var(--colors-background);
+    border-top: 1px solid var(--colors-border);
+  `,
+  warning: css`
+    color: var(--colors-warning-text);
+    background-color: var(--colors-warning-background);
+    border-top: 1px solid var(--colors-border);
+  `,
+  // default variant for title
+  title: css`
+    color: var(--colors-primary);
+    background-color: var(--colors-secondaryBackground);
+  `,
+  // variants for Overlay Banner
+  banner: css`
+    color: var(--colors-primary);
+    background-color: var(--colors-background);
+  `,
+  alignright: css`
```

Comment:
Is `alignrightbanner` okay? Can also do `actionsbanner` because it would only be used there. I'm thrown errors when I try to hyphenate and I'm not sure if camel case is the convention here.

Reply:
ohhh how about `actionsbanner` and `infobanner` under `sectionVariants` ?